sdcm/nemesis.py

```diff
@@ -2964,8 +2966,53 @@ def get_rate_limit_for_network_disruption(self) -> Optional[str]:
 
         return "{}{}".format(random.randrange(min_limit, max_limit), rate_limit_suffix)
 
+    def _disrupt_network_random_interruptions_k8s(self, list_of_timeout_options):
+        interruptions = ["delay", "loss", "corrupt"]
+        rate_limit: Optional[str] = self.get_rate_limit_for_network_disruption()
+        if not rate_limit:
+            self.log.warning("NetworkRandomInterruption won't limit network bandwidth due to lack of monitoring nodes.")
+        else:
+            interruptions.append("rate")
+        duration = f"{random.choice(list_of_timeout_options)}s"
+        match random.choice(interruptions):
+            case "delay":
+                delay_in_msecs = random.randrange(50, 300)
```

Comment:
Let's use randomness based on the nemesis seed, so we can reproduce those ones ?

Reply:
It's not that trivial, these nemeses all use `random` without seed in every place (not just delay, but also which interruption and associated parameters), doing it in just one would not make much sense.

I can add followup task as anyway would be better to do in separate commit.

Reply:
I agree with @soyacz the change of that approach is not target of this PR. It can and must be done separately.

Reply:
It should be the approach on any random call we add to a nemesis.

I know it's not the case in all of the nemesis right now, but we should strive doing such when we add new ones, or touch existing ones.

I don't think it's such a big change for this PR.

But a followup to change them all to be based on the nemesis seed is also o.k.
If that's the case please open a task/issue for that